### ISSUE TITLE
Research Question

### ISSUE THREAD

**Issue by frabbitry** [robftg/Rocket-Launches#13]
@biancamaginley @robftg @BMT45 

Hi guys. I think Brad talked about some of this stuff on the meeting on Friday, but I think that now would be a good time for you guys to start thinking about what the research question for the project will be. At the very least, this will give you guys something to think about along with the website (and it should inform your website design, too; design should support content, and the research question will dictate your content). 

You've got data, now. What do you want to find out from the data? What should your graphs eventually show? Remembering your [project proposal](https://github.com/ebeshero/DHClass-Hub/issues/745), I think that you guys were originally interested in kind of comparing and contrasting government launches vs. private ones. What kinds of vehicles are used in government launches versus private ones? I think that Brad talked to you guys about correlating launch time, duration, and inclination/altitude. 

In short, what question(s) do you want to ask of your data? What do you want to graph? Maybe you guys can brainstorm some ideas and comment on this issue with them. If you guys have any questions, feel free to ping me, or I'll be at class today. I just want to know what you guys are going to be aiming for in this second phase of the project: data visualization.
